cython_lint/cython_lint.py

```diff
@@ -862,18 +902,27 @@ def main(argv: Sequence[str] | None = None) -> int:  # pragma: no cover
         help='Comma-separated list of pycodestyle error codes to ignore',
     )
     args = parser.parse_args(argv)
+    paths = [pathlib.Path(path) for path in args.paths]
+
+    # Update defaults from pyproject.toml or setup.cfg if present
+    config = {k.replace('-', '_'): v for k, v in _get_config(paths).items()}
```

Comment:
can we keep this simple and require that the tool be run from the root of the repository?

Reply:
actually, I guess this is fine, it's a fairly common thing to support